### ISSUE TITLE
Accessibility Tester Report

### ISSUE THREAD

**Issue by tonifisler** [vdch/regenair-fabricator#621]
Accessibility is tested with [Accessibility Developer Tools](https://chrome.google.com/webstore/detail/accessibility-developer-t/fpkknkljclfencbdbgkenhalefipecmb/related?hl=en)

### [Warning] Text elements should have a reasonable contrast ratio (4)
See [AX_COLOR_01](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_color_01) for more information.

**Footer:**
![screen shot 2017-03-09 at 09 57 48](https://cloud.githubusercontent.com/assets/1708450/23742964/35c118ae-04af-11e7-8ee5-65176e60c00e.png)

### [Warning] The purpose of each link should be clear from the link text (5)
See [AX_TEXT_04](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_text_04) for more information.

**Article teaser**
![screen shot 2017-03-09 at 10 05 01](https://cloud.githubusercontent.com/assets/1708450/23743169/f2d84ebc-04af-11e7-82f3-aa4a5a84afd7.png)

### [Warning] These elements are focusable but either invisible or obscured by another element (26)
See [AX_FOCUS_01](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules#ax_focus_01) for more 

(a lot of elements)

### Several ARIA issues
- AX_ARIA_02
- AX_ARIA_04
- AX_ARIA_10
